.github/CODEOWNERS

```diff
@@ -1044,6 +1044,24 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 
 
 # Data Discovery
+/test/functional/fixtures/kbn_archiver/dashboard_error_cases.json @elastic/kibana-data-discovery
```

Comment:
I think this one should be assigned to @elastic/kibana-presentation

Reply:
https://github.com/elastic/kibana/pull/200932/commits/a92e6fee77875fa5ddc26c0f1c523151b0fead41